doc/adr/0006-url-format-in-qr-code.md

```diff
@@ -0,0 +1,44 @@
+# 6. URL format in QR code
+
+Date: 2022-06-24
+
+## Status
+
+Accepted
+
+## Context
+
+The web widget displays a QR code, which transfers data to the mobile device, i.e. into the eID client.
+
+The QR code needs to include the ClientURL, including the `tcTokenURL`, (see [TR-03124 part 1](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Publikationen/TechnischeRichtlinien/TR03124/TR-03124-1.pdf) for more information).
+
+Additionally, some kind of widget session id needs to be added, so that the proxy message server can identify this exact widget when the mobile app wants to send a message to the widget.
+
+Lastly, an [encryption key](https://github.com/digitalservicebund/useid-architecture/blob/50b667dfe5d38846f3ac46ae8ab2a69c2ed63afa/doc/adr/0004-e2ee-for-refresh-address.md) needs to added. This should never be sent to any server and only be known to the widget and the mobile app.
```

Comment:
The link should be updated after [useid-architecture#5](https://github.com/digitalservicebund/useid-architecture/pull/5) was merged.